src/swsssdk/dbconnector.py

```diff
@@ -27,91 +93,136 @@ def load_sonic_db_config(sonic_db_file_path=SONIC_DB_CONFIG_FILE):
                 logger.warning(msg)
                 sonic_db_file_path = os.path.join(os.path.dirname(os.path.abspath(__file__)), 'config', 'database_config.json')
             with open(sonic_db_file_path, "r") as read_file:
-                SonicDBConfig._sonic_db_config = json.load(read_file)
+                # The database_config.json is loaded into the '' index, which referes to the local namespace.
```

Comment:
'' index seems a little bit confusing.

Reply:
Updated based on discussion to continue using '' internally but externally the user will specify namespace=None, when he wants to connect to local instances.